### PR TITLE
feat(v4): make aside container

### PR DESCRIPTION
**Pull request by Haroenv**

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -240,23 +240,15 @@ mark {
...
-}
-
-@keyframes fadeIn {
-  from {
-    opacity: 0;
-  }
-  to {
-    opacity: 1;
-  }
+  height: 128px;
```
Comment (francoischalifour) [inline src/app.css]: This distorts images though, so not sure if better. I think we can remove the two following lines.

File: src/app.js [hunk 2]
```diff
@@ -28,16 +28,16 @@ const structuredResults = connectHits(({ hits, widgetParams }) => {
...
+      <img class="image" src="${result.url}" alt=${result.name} />
+      <h3>${result.name}</h3>
     `;
     return;
   }
 
-  container.innerHTML =
-    '<p>No structured results for this query – try e.g. "Acer".</p>';
+  container.innerHTML = `
+    <div class="image"></div>
```
Comment (francoischalifour) [inline src/app.js]: I think it's OK to remove this. It's weirder than having a different height IMO.

File: src/app.css (hunk 1 above)
Comment (Haroenv) [reply src/app.css]: I only give it a height, the width is flexible no?

File: src/app.css [hunk 3]
```diff
@@ -249,14 +249,5 @@ mark {
 
 #structured-result img {
   margin: 1rem auto;
-  animation: fadeIn 150ms;
-}
-
-@keyframes fadeIn {
```
Comment (Haroenv) [inline src/app.css]: this didn't do anything, since an animation doesn't run in this case

**Pull request merged**